jupyter-notebooks/Supplemental_PAO1_FastICA_example.ipynb

```diff
@@ -0,0 +1,1467 @@
+      "text/plain": [
+       "<matplotlib.figure.Figure at 0x1d340f487f0>"
```

Comment:
Not sure what I am supposed to get from this figure? that all the distributions are kinda similar? It may be worth to add a sentence or two of your interpretation